### ISSUE TITLE
[Bug]: Seekbar preview thumbnail blocks input

### ISSUE THREAD

**Issue by Nobylo** [FreeTubeApp/FreeTube#5965]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Open a video and wait for the player to start playing in windowed mode
2. Position mouse below video player
3. Move mouse straight up, right above the seek bar and immediately press the left mouse button to pause
(Mouse will have stopped and clicked on the seekbar preview thumbnail, player is not paused. Behavior can be inconsistent)

### Expected Behavior

Video pauses

### Issue Labels

inconsistent behavior, usability issue

### FreeTube Version

v0.22.0, 30c3978

### Operating System Version

Windows 11 Build 22631

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.21.3, 8b66548

### Nightly Build

- [x] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@Nobylo could you provide a screenrecording?

**Comment by geominorai** [comment]
This also affects using the mouse wheel to skip through the video feature.  Instead of skipping, it causes the entire page to scroll down instead.

It is especially obvious with shorts because the thumbnail for those are much larger, so the chance that moving the mouse from below and over the video player ends up landing on it and triggering this bug.

**Comment by geominorai** [comment]
Here are video recordings of the thumbnail interfering with scrolling and clicking:

https://github.com/user-attachments/assets/7a2b88e1-e4b4-4e6f-9213-0c7b47d769d8

https://github.com/user-attachments/assets/5aa6b6de-fcf8-436c-b4db-1106810e6932

**Comment by absidue** [comment]
You are saying that you want to be able to click and scroll on the thumbnail preview, while it is visible and want your scroll and click event to go to somewhere else than the thing you actually clicked on/scrolled over?

Could someone please clarifiy if my understanding of what you are saying is correct?

**Comment by geominorai** [comment]
Yes, clicking and scrolling over the area within the thumbnail preview should act on whatever was underneath it as if the thumbnail preview never appeared.  This is the case on the YouTube website's video player, where even if you move the mouse cursor fast enough to catch the thumbnail, clicking on it just pauses/resumes the video.

On FreeTube, the thumbnail preview is briefly focusable before it fades away, which even makes it possible to accidentally select the image and timestamp text if you happen to click and drag over it:

![image](https://github.com/user-attachments/assets/c83bdfb7-5551-4743-a8d6-8695eec1d129)

**Comment by Nobylo** [comment]
Here is more information about the inconsistent behavior mentioned originally:

If I do the motion from the original post slowly, or otherwise try to bring up the preview from above, it will appear then disappear after around 300ms. It'll block mouse clicks within those 300ms. There's no reason to wanna click the preview in this state. I have to move my mouse around the preview or otherwise wait for it to disappear if I wanna pause.

If I move the mouse faster, something happens (sometimes) that makes the preview window stay until I move the mouse off it. This is the source of the inconsistent behavior. I don't know which of these 2 behaviors are intended.

Here is the test footage I captured, it showcases both behaviors:

https://github.com/user-attachments/assets/34526cc7-cd75-4dbb-a25c-e2d89185b4d9

(This was tested and recorded on nightly [5cd4981](https://github.com/FreeTubeApp/FreeTube/commit/5cd4981af48bd3609706fca287bc68e6fe5294bc))

Thank you @geominorai for providing screen recordings when I couldn't. I've been having issues with github notifications recently. The behavior you are experiencing differs from mine in that, clicking does not cause it to stay for me and I can't drag the preview image around like you can. I can only highlight and copy the preview's timestamp whenever it stays. What operating system you are running? (I'm guessing that would be the reason for the differences)

**Comment by geominorai** [comment]
> Thank you @geominorai for providing screen recordings when I couldn't. I've been having issues with github notifications recently. The behavior you are experiencing differs from mine in that, clicking does not cause it to stay for me and I can't drag the preview image around like you can. I can only highlight and copy the preview's timestamp whenever it stays. What operating system you are running? (I'm guessing that would be the reason for the differences)

@Nobylo: I am on Windows 10 and I think our bugs are the same, as I do not have to click for the thumbnail to stay up.  It is just that I can click on it while it stays up from hovering over it.  After that if we start the drag-select over the text then continue over the thumbnail image, all that gets highlighted and then can be dragged like in my screenshot.  It is just meant to demonstrate that it steals focus.

For activating the bug in the first place, my guess is depending on the speed of the mouse cursor, there seems to be a slight gap between the seek bar and the preview thumbnail area that if the cursor happens to land on it briefly, it will cause trigger the preview thumbnail to hide shortly afterwards, but if the cursor is too fast then it skips right past it.

**Comment by PhilJbt** [comment]
I have the same problem. I sometimes have to try several times to pause a video. The preview delay matches the time it takes me to move the cursor to the Freetube window.

Remove any click/scroll event of the preview would fix that, It doesn't seem to me that there is any interaction with it anyway.

[2025-01-12 16-08-50.webm](https://github.com/user-attachments/assets/a18cdcb5-8c07-4f91-a3a7-1a8a2c35b8d8)

**Comment by kommunarr** [comment]
I can replicate this. The issue seems to be that the seek bar preview thumbnail stays visible on hover of the thumbnail, when it should in fact disappear in such a case (e.g., how YT handles it). Not sure if we want to mark this one as an upstream issue or not. CC: @absidue

**Comment by PikachuEXE** [comment]
A quick update:
Fix on upstream merged, pending release and FT has regular update PRs to get new versions
So just need to wait..
https://github.com/shaka-project/shaka-player/pull/8089

**Comment by geominorai** [comment]
Fantastic news, thanks!

**Comment by Nobylo** [comment]
I've been too busy to get back to this
Note the orange gap - behavior is a lot more predictable now but it's still "inconsistent"
![Image](https://github.com/user-attachments/assets/51bdfe9f-ab98-4919-96d2-a044ec1edf8d)

I did a terrible job explaining cause my issue was never inconsistent behavior from the preview and was always "why can you click the preview at all" with how it causes this problem:
![Image](https://github.com/user-attachments/assets/34b7c423-936c-4e04-a004-fdd1e42f12c9)
This entire quarter of the video player is a zone you're not allowed to click in if you wanna know that your click is gonna pause the video. The problem is also helped a bit by shaka's big UI but that's not changing any time soon from what I got

JUST LEAVE IT it's been 4 months I'll look into external players I'm just one guy

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@Nobylo this should be fixed and will be included in the next minor release

**Comment by Nobylo** [comment]
> [@Nobylo](https://github.com/Nobylo) this should be fixed and will be included in the next minor release

Oh it is done? Some day I will get a clue about github, for now do not mind the old man yelling at clouds 😅
